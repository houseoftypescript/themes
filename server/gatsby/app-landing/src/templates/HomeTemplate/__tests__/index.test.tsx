import { render } from '@testing-library/react';
import { HomeTemplate } from '../index';
import React from 'react';

describe('HomeTemplate', () => {
  it('to match snapshot', () => {
    const { container } = render(<HomeTemplate />);
    expect(container).toMatchSnapshot();
  });
});
