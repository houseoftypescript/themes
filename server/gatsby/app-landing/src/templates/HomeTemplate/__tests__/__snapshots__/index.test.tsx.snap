// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeTemplate to match snapshot 1`] = `
<div>
  <div
    class="gradient leading-normal tracking-normal text-white"
  >
    <nav
      class="fixed top-0 z-50 w-full "
      id="header"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="py-8"
        >
          <div
            class="flex w-full items-center justify-between"
          >
            <a
              class="flex items-center gap-2 text-white"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge fill-current css-tzssek-MuiSvgIcon-root"
                data-testid="FlightTakeoffIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M2.5 19h19v2h-19v-2zm19.57-9.36c-.21-.8-1.04-1.28-1.84-1.06L14.92 10l-6.9-6.43-1.93.51 4.14 7.17-4.97 1.33-1.97-1.54-1.45.39 2.59 4.49s7.12-1.9 16.57-4.43c.81-.23 1.28-1.05 1.07-1.85z"
                />
              </svg>
              <span
                class="text-xl font-bold md:text-2xl"
              >
                LANDING
              </span>
            </a>
            <div
              class="hidden md:block text-white"
            >
              <div
                class="flex items-center gap-8"
              >
                <a
                  class="cursor-pointer font-medium capitalize tracking-wider"
                >
                  Services
                </a>
                <a
                  class="cursor-pointer font-medium capitalize tracking-wider"
                >
                  projects
                </a>
                <a
                  class="cursor-pointer font-medium capitalize tracking-wider"
                >
                  pricing
                </a>
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-q23ilp-MuiButtonBase-root-MuiButton-root"
                  tabindex="0"
                  type="button"
                >
                  Download
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
            <div
              class="block md:hidden"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium text-white css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="MenuIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </nav>
    <section
      id="hero"
    >
      <div
        class="pt-32"
      >
        <div
          class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
        >
          <div
            class="grid grid-cols-1 gap-8 md:grid-cols-2"
          >
            <div
              class="col-span-1"
            >
              <div
                class="flex h-full items-center"
              >
                <div
                  class="flex w-full flex-col text-center md:text-left"
                >
                  <p
                    class="tracking-loose w-full uppercase"
                  >
                    What business are you?
                  </p>
                  <h1
                    class="my-4 text-5xl font-bold leading-tight"
                  >
                    Main Hero Message to sell yourself!
                  </h1>
                  <p
                    class="mb-8 text-2xl leading-normal"
                  >
                    Sub-hero message, not too long and not too short. Make it just right!
                  </p>
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium mx-auto w-32 md:mx-0 css-mdb7lr-MuiButtonBase-root-MuiButton-root"
                    tabindex="0"
                    type="button"
                  >
                    Download
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
            </div>
            <div
              class="col-span-1"
            />
          </div>
        </div>
      </div>
      <div />
    </section>
    <section
      class="bg-white py-8 md:py-16"
      id="services"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="mb-8 flex flex-col items-center justify-center"
        >
          <h1
            class="mb-2 w-full text-center text-5xl font-bold leading-tight text-white text-gray-900"
          >
            Services
          </h1>
          <div
            class="w-full"
          >
            <div
              class="gradient mx-auto h-1 w-64 rounded-t bg-white py-0 opacity-25"
            />
          </div>
        </div>
        <div
          class="flex flex-col gap-8"
        >
          <div
            class="grid grid-cols-1 gap-8 md:grid-cols-2"
          >
            <div
              class="order-2 col-span-1 md:order-1"
            >
              <div
                class="flex h-full items-center"
              >
                <div
                  class="flex flex-col gap-4"
                >
                  <h2
                    class="text-3xl font-bold text-gray-900"
                  >
                    Lorem ipsum dolor sit amet
                  </h2>
                  <p
                    class="text-gray-700"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at ipsum eu nunc commodo posuere et sit amet ligula.
                  </p>
                </div>
              </div>
            </div>
            <div
              class="order-1 col-span-1 md:order-2"
            />
          </div>
          <div
            class="grid grid-cols-1 gap-8 md:grid-cols-2"
          >
            <div
              class="col-span-1"
            />
            <div
              class="col-span-1"
            >
              <div
                class="flex h-full items-center"
              >
                <div
                  class="flex flex-col gap-4"
                >
                  <h2
                    class="text-3xl font-bold text-gray-900"
                  >
                    Lorem ipsum dolor sit amet
                  </h2>
                  <p
                    class="text-gray-700"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at ipsum eu nunc commodo posuere et sit amet ligula.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section
      class="bg-white py-8 md:py-16"
      id="projects"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="mb-8 flex flex-col items-center justify-center"
        >
          <h1
            class="mb-2 w-full text-center text-5xl font-bold leading-tight text-white text-gray-900"
          >
            Projects
          </h1>
          <div
            class="w-full"
          >
            <div
              class="gradient mx-auto h-1 w-64 rounded-t bg-white py-0 opacity-25"
            />
          </div>
        </div>
        <div
          class="grid grid-cols-1 gap-8 md:grid-cols-3"
        >
          <div
            class="col-span-1"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root border shadow css-pwngrv-MuiPaper-root-MuiCard-root"
            >
              <div
                class="MuiCardContent-root px-8 pt-8 pb-4 css-46bh2p-MuiCardContent-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-16m3jxf-MuiTypography-root"
                >
                  Project #
                  1
                </p>
                <div
                  class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
                >
                  Lorem ipsum dolor sit amet.
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body1 css-9eg9l8-MuiTypography-root"
                >
                  getting started
                </p>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
                >
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at ipsum eu nunc commodo posuere et sit amet ligula.
                </p>
              </div>
              <div
                class="MuiCardActions-root MuiCardActions-spacing px-8 pb-8 pt-4 css-1t6e9jv-MuiCardActions-root"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1m0onwi-MuiButtonBase-root-MuiButton-root"
                  tabindex="0"
                  type="button"
                >
                  Learn More
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
          </div>
          <div
            class="col-span-1"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root border shadow css-pwngrv-MuiPaper-root-MuiCard-root"
            >
              <div
                class="MuiCardContent-root px-8 pt-8 pb-4 css-46bh2p-MuiCardContent-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-16m3jxf-MuiTypography-root"
                >
                  Project #
                  2
                </p>
                <div
                  class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
                >
                  Lorem ipsum dolor sit amet.
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body1 css-9eg9l8-MuiTypography-root"
                >
                  getting started
                </p>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
                >
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at ipsum eu nunc commodo posuere et sit amet ligula.
                </p>
              </div>
              <div
                class="MuiCardActions-root MuiCardActions-spacing px-8 pb-8 pt-4 css-1t6e9jv-MuiCardActions-root"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1m0onwi-MuiButtonBase-root-MuiButton-root"
                  tabindex="0"
                  type="button"
                >
                  Learn More
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
          </div>
          <div
            class="col-span-1"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root border shadow css-pwngrv-MuiPaper-root-MuiCard-root"
            >
              <div
                class="MuiCardContent-root px-8 pt-8 pb-4 css-46bh2p-MuiCardContent-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-16m3jxf-MuiTypography-root"
                >
                  Project #
                  3
                </p>
                <div
                  class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
                >
                  Lorem ipsum dolor sit amet.
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body1 css-9eg9l8-MuiTypography-root"
                >
                  getting started
                </p>
                <p
                  class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
                >
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at ipsum eu nunc commodo posuere et sit amet ligula.
                </p>
              </div>
              <div
                class="MuiCardActions-root MuiCardActions-spacing px-8 pb-8 pt-4 css-1t6e9jv-MuiCardActions-root"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1m0onwi-MuiButtonBase-root-MuiButton-root"
                  tabindex="0"
                  type="button"
                >
                  Learn More
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section
      class="bg-gray-100 py-8 md:py-16"
      id="pricing"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="mb-8 flex flex-col items-center justify-center"
        >
          <h1
            class="mb-2 w-full text-center text-5xl font-bold leading-tight text-white text-gray-900"
          >
            Pricing
          </h1>
          <div
            class="w-full"
          >
            <div
              class="gradient mx-auto h-1 w-64 rounded-t bg-white py-0 opacity-25"
            />
          </div>
        </div>
        <div
          class="grid grid-cols-1 gap-8 md:grid-cols-3"
        >
          <div
            class="col-span-1"
          >
            <div
              class="rounded border bg-white shadow"
            >
              <div
                class="py-8 text-center"
              >
                <h2
                  class="text-3xl font-bold text-gray-700"
                >
                  Free
                </h2>
              </div>
              <div
                class="gradient h-1 w-full"
              />
              <div
                class="text-gray-500"
              >
                <div
                  class="border-b py-4 text-center capitalize"
                >
                  feature 1
                </div>
                <div
                  class="border-b py-4 text-center capitalize"
                >
                  feature 2
                </div>
                <div
                  class="border-b py-4 text-center capitalize"
                >
                  feature 3
                </div>
                <div
                  class="border-b py-4 text-center capitalize"
                >
                  feature 4
                </div>
              </div>
              <div
                class="flex flex-col gap-4 py-8"
              >
                <h3
                  class="text-center text-3xl text-gray-900"
                >
                  $
                  0
                </h3>
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium mx-auto css-1m0onwi-MuiButtonBase-root-MuiButton-root"
                  tabindex="0"
                  type="button"
                >
                  Download
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
          </div>
          <div
            class="col-span-1"
          >
            <div
              class="rounded border bg-white shadow"
            >
              <div
                class="py-8 text-center"
              >
                <h2
                  class="text-3xl font-bold text-gray-700"
                >
                  Basic
                </h2>
              </div>
              <div
                class="gradient h-1 w-full"
              />
              <div
                class="text-gray-500"
              >
                <div
                  class="border-b py-4 text-center capitalize"
                >
                  feature 5
                </div>
                <div
                  class="border-b py-4 text-center capitalize"
                >
                  feature 6
                </div>
                <div
                  class="border-b py-4 text-center capitalize"
                >
                  feature 7
                </div>
                <div
                  class="border-b py-4 text-center capitalize"
                >
                  feature 8
                </div>
              </div>
              <div
                class="flex flex-col gap-4 py-8"
              >
                <h3
                  class="text-center text-3xl text-gray-900"
                >
                  $
                  9.99
                </h3>
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium mx-auto css-1m0onwi-MuiButtonBase-root-MuiButton-root"
                  tabindex="0"
                  type="button"
                >
                  Download
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
          </div>
          <div
            class="col-span-1"
          >
            <div
              class="rounded border bg-white shadow"
            >
              <div
                class="py-8 text-center"
              >
                <h2
                  class="text-3xl font-bold text-gray-700"
                >
                  Pro
                </h2>
              </div>
              <div
                class="gradient h-1 w-full"
              />
              <div
                class="text-gray-500"
              >
                <div
                  class="border-b py-4 text-center capitalize"
                >
                  feature 9
                </div>
                <div
                  class="border-b py-4 text-center capitalize"
                >
                  feature 10
                </div>
                <div
                  class="border-b py-4 text-center capitalize"
                >
                  feature 11
                </div>
                <div
                  class="border-b py-4 text-center capitalize"
                >
                  feature 12
                </div>
              </div>
              <div
                class="flex flex-col gap-4 py-8"
              >
                <h3
                  class="text-center text-3xl text-gray-900"
                >
                  $
                  19.99
                </h3>
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium mx-auto css-1m0onwi-MuiButtonBase-root-MuiButton-root"
                  tabindex="0"
                  type="button"
                >
                  Download
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section
      id="cta"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="pb-16 md:pb-32"
        >
          <div
            class="mb-8 flex flex-col items-center justify-center"
          >
            <h1
              class="mb-2 w-full text-center text-5xl font-bold leading-tight text-white text-white"
            >
              Call To Action
            </h1>
            <div
              class="w-full"
            >
              <div
                class="bg-white mx-auto h-1 w-64 rounded-t bg-white py-0 opacity-25"
              />
            </div>
          </div>
          <h2
            class="mb-8 text-center text-3xl leading-tight"
          >
            Main Hero Message to sell yourself!
          </h2>
          <div
            class="flex"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge mx-auto css-5zr34g-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              Download
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </section>
    <footer
      class="bg-white"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="py-8"
        >
          <div
            class="flex justify-between text-pink-600"
          >
            <a
              class="flex gap-2"
              href="/#"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium fill-current css-i4bv87-MuiSvgIcon-root"
                data-testid="FlightTakeoffIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M2.5 19h19v2h-19v-2zm19.57-9.36c-.21-.8-1.04-1.28-1.84-1.06L14.92 10l-6.9-6.43-1.93.51 4.14 7.17-4.97 1.33-1.97-1.54-1.45.39 2.59 4.49s7.12-1.9 16.57-4.43c.81-.23 1.28-1.05 1.07-1.85z"
                />
              </svg>
              <span
                class="text-lg font-bold md:text-xl"
              >
                LANDING
              </span>
            </a>
            <div
              class="flex items-center gap-2 md:gap-4"
            >
              <a
                href="https://facebook.com"
                target="_blank"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium fill-current css-i4bv87-MuiSvgIcon-root"
                  data-testid="FacebookIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2m13 2h-2.5A3.5 3.5 0 0 0 12 8.5V11h-2v3h2v7h3v-7h3v-3h-3V9a1 1 0 0 1 1-1h2V5z"
                  />
                </svg>
              </a>
              <a
                href="https://twitter.com"
                target="_blank"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium fill-current css-i4bv87-MuiSvgIcon-root"
                  data-testid="TwitterIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M22.46 6c-.77.35-1.6.58-2.46.69.88-.53 1.56-1.37 1.88-2.38-.83.5-1.75.85-2.72 1.05C18.37 4.5 17.26 4 16 4c-2.35 0-4.27 1.92-4.27 4.29 0 .34.04.67.11.98C8.28 9.09 5.11 7.38 3 4.79c-.37.63-.58 1.37-.58 2.15 0 1.49.75 2.81 1.91 3.56-.71 0-1.37-.2-1.95-.5v.03c0 2.08 1.48 3.82 3.44 4.21a4.22 4.22 0 0 1-1.93.07 4.28 4.28 0 0 0 4 2.98 8.521 8.521 0 0 1-5.33 1.84c-.34 0-.68-.02-1.02-.06C3.44 20.29 5.7 21 8.12 21 16 21 20.33 14.46 20.33 8.79c0-.19 0-.37-.01-.56.84-.6 1.56-1.36 2.14-2.23z"
                  />
                </svg>
              </a>
              <a
                href="https://linkedin.com"
                target="_blank"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium fill-current css-i4bv87-MuiSvgIcon-root"
                  data-testid="LinkedInIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                  />
                </svg>
              </a>
            </div>
          </div>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
